src/assets/pos.ts

```diff
@@ -206,12 +291,12 @@ export const createGenesisDataObj = async (
 	const topValidators = voteWeightR2.delegates;
 
 	const initValidators: Buffer[] = [];
-	const accountbannedMap = new Map(
+	const accountBannedMap = new Map(
```

Comment:
minor point but probably we could create the map somewhere outside and reuse in multiple places.